corehq/apps/README.rst

```diff
@@ -0,0 +1,190 @@
+   Most of this app relies heavily on standard reporting functionality.
+fixtures
+   "Fixtures" is used generally in the codebase to refer to datasets sent to the phone, which includes the
+   casedb, mobile report data, etc., but the fixtures app primarily deals with one specific type of fixture,
+   which is lookup tables.
+groups
+   Users can be assigned to groups for the purposes of sharing cases within a group and for reporting purposes.
+hqadmin
+   Internal admin functionality used by the development, support, QA, and product teams.
+linked_domain
+   Functionality to share certain configuration data between domains: apps, lookup tables, report definitions, etc.
+   Work is done in a primary "upstream" domain, and then that domain's data models are copied to one or more
+   "downstream" domains. This is most often used to set up a development => production workflow, where changes are made
+   in the development domain and then pushed to the production domain, where is where real project data is entered.
+   Linekd domains are also used by certain enterprise-type projects that manage one program across multiple regions
+   and use a separate downstream domain for each region.
+registration
+   Workflows for creating new accounts on HQ.
+reminders
+   A subset of SMS/messaging, including functionality around incoming SMS keywords. "Reminders" is a leftover term from a previous iteration of the messaging framework.
+saved_reports
+   Functionality to let users save a specific set of report filters and optionally run reports with those filters on a scheduled basis.
+toggle_ui
+   Framework for feature flags, which are used to limit internatl feature to specific domains and/or users.
+translations
+   Functionality for managing application translations, including integration with Transifex, which is used by a small number of projects.
+user_importer
+   Bulk importing of users.
+
+Tertiary Apps
+^^^^^^^^^^^^^
+These apps may be useful parts of the system but don't have as much active development as the groups above.
+
+aggregate_ucrs
+   TODO
+analytics
+   Integrations with third-party analytics tools such as Google Analytics and Kissmetrics.
+   Also contains internal product-focused tools such as AB testing functionality.
+builds
+   Models relating to CommCare Mobile builds, so that app builders can control which mobile version their apps use.
+case_search
+   TODO
+dashboard
+   The tiled UI that acts as the main landing page for HQ.
+formplayer_api
+   Functionality interacting with formplayer, primarily used by SMS surveys.
+mobile_auth
+   Generates the XML needed to authorize mobile users.
+notifications
+   "Banner" notifications used by the support team to notify users of upcoming downtime,
+   ongoing issues, etc.
+ota
+   Functionality at the interface of CommCare HQ and CommCare Mobile: demo users, device logs, mobile recovery, etc.
+receiverwrapper
+   TODO
```

Comment:
@mkangia or @AmitPhulera or @snopoke  Can you replace this with a sentence?

Reply:
```suggestion
   Contains the API for receiving XML form submissions. This app mostly deals with the interfacing portion of the
   API including auth, rate limiting etc. but not the actual data processing which is contained in the
   `form_processor` app.
```